client/my-sites/plans/plan-overview/style.scss

```diff
@@ -0,0 +1,27 @@
+@mixin plan-overview-button {
```

Comment:
@scruffian it looks like we should use @extend for mixins that don't accept an argument ([docs](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#mixins-vs-extends)).